src/types/Status.ts

```diff
@@ -0,0 +1,5 @@
+export enum Status {
+  all = 'all',
+  active = 'active',
+  completed = 'completed',
+}
```

Comment:
It is customary to write the names of enum values either in all capital letters or with a capital letter
```suggestion
export enum Status {
  All = 'all',
  Active = 'active',
  Completed = 'completed',
}
```